pkg/cmd/tknpac/bootstrap/bootstrap.go

```diff
@@ -68,14 +69,14 @@ var successTmpl = fmt.Sprintf(`
 
 func install(ctx context.Context, run *params.Run, opts *bootstrapOpts) error {
 	if !opts.forceInstall {
-		fmt.Fprintln(opts.ioStreams.Out, "🏃 Checking if Pipelines as Code is installed.")
+		fmt.Fprintln(opts.ioStreams.Out, "=> Checking if Pipelines as Code is installed.")
 	}
 	tektonInstalled, err := checkPipelinesInstalled(run)
 	if err != nil {
 		return err
 	}
 	if !tektonInstalled {
-		return fmt.Errorf("a Tekton installation has not been found on this cluster, install Tekton first before launching this command")
+		return fmt.Errorf("no Tekton installation has been found on this kubernetes cluster, install Tekton first before launching this command")
```

Comment:
does `kubernetes` word on Openshift cluster won't mislead :thinking:

Reply:
i think that's fine :) most cusotmers now when they running openshift they run kube